topics/single-cell/tutorials/scrna-case_FilterPlotandExplore_SeuratTools/tutorial.md

```diff
@@ -0,0 +1,763 @@
+
+Now what about the opposite end of the spectrum: the double negative early T-cells? We can see from our table about that Il2ra is a known marker of this cell type, let's see where it's most strongly expressed in the data: 
+
+> <hands-on-title>Plot Il2ra </hands-on-title>
+>
+> Run{% tool [Plot with Seurat](toolshed.g2.bx.psu.edu/repos/ebi-gxa/seurat_plot/seurat_plot/4.0.4+galaxy0) %} with the following parameters:
+> - *"RDS file"*: `Seurat UMAP on Data 20: Seurat RDS`
+> - *"Plot_type_selector"*: `FeaturePlot`
+> - *"Feature"*: `Il2ra`
+{: .hands_on}
+
+![FeaturePlot Il2ra](../../images/scrna-case_FPE_SeuratTools/FeaturePlot_Il2ra.png "FeaturePlot of Il2ra")
+
+Il2ra expression seems to be most prominent in Cluster 2 and as seen in our marker list, has a fold change of 2.79! 
+
+It can, and should, act as a sanity check that the most (and least) differentiated cell types expected in the data appear as two "island" clusters. 
+
+Now for the intermediate populations--which may be a bit more tricky to deconvolute. Based on our known markers, we see that both the double-positive populations express many of the same genes: Cd8b1, Cd8a, and Cd4. Let's make sure that the remaining clusters (0, 1, 3, 4, 5, 6, and 7) all express these: 
+
+> <hands-on-title>Plot Cd8b1 </hands-on-title>
+>
+> Run{% tool [Plot with Seurat](toolshed.g2.bx.psu.edu/repos/ebi-gxa/seurat_plot/seurat_plot/4.0.4+galaxy0) %} with the following parameters:
+> - *"RDS file"*: `Seurat UMAP on Data 20: Seurat RDS`
+> - *"Plot_type_selector"*: `FeaturePlot`
+> - *"Feature"*: `Cd8b1`
+{: .hands_on}
+
+![FeaturePlot Cd8b1](../../images/scrna-case_FPE_SeuratTools/FeaturePlot_Cd8b1.png "FeaturePlot of Cd8b1")
+
+It looks like clusters 1, 4, 5, and 6 pretty strongly express Cd8b1, now what about Cd8a? 
+
+> <hands-on-title>Plot Cd8a </hands-on-title>
+>
+> Run{% tool [Plot with Seurat](toolshed.g2.bx.psu.edu/repos/ebi-gxa/seurat_plot/seurat_plot/4.0.4+galaxy0) %} with the following parameters:
+> - *"RDS file"*: `Seurat UMAP on Data 20: Seurat RDS`
+> - *"Plot_type_selector"*: `FeaturePlot`
+> - *"Feature"*: `Cd8a`
+{: .hands_on}
+
+![FeaturePlot Cd8a](../../images/scrna-case_FPE_SeuratTools/FeaturePlot_Cd8a.png "FeaturePlot of Cd8a")
+
+This looks pretty consistent with the Cd8b1 plot, which is expected as these are markers of a double positive population! The true discernment between the two populations will be the level of Cd4 expression: 
+
+> <hands-on-title>Plot Cd4 </hands-on-title>
+>
+> Run{% tool [Plot with Seurat](toolshed.g2.bx.psu.edu/repos/ebi-gxa/seurat_plot/seurat_plot/4.0.4+galaxy0) %} with the following parameters:
+> - *"RDS file"*: `Seurat UMAP on Data 20: Seurat RDS`
+> - *"Plot_type_selector"*: `FeaturePlot`
+> - *"Feature"*: `Cd4`
+{: .hands_on}
+
+![FeaturePlot Cd4](../../images/scrna-case_FPE_SeuratTools/FeatuerePlot_Cd4.png "FeaturePlot of Cd4")
```

Comment:
It looks like this alt-text doesn't describe the image very well for users who are relying on a screen reader to access the content. We have a short guide on [what's appropriate for alt-text vs captions in the GTN Contributing Guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text) which has more details. *This comment may apply to more images, please consider checking all of them.*